composer.json

```diff
@@ -12,8 +12,8 @@
     "require": {
         "php": ">=5.3.3",
         "mapbender/mapbender": "^3.2.6",
-        "mapbender/data-manager": "^1.2.2 || ^2",
```

Comment:
same here. I am aware that we have to deal with cross-dependencies here, what may work is to require a version that is yet to be released and then release new versions of all three packages simultaneously